genocrowd/react/src/routes/error/error.jsx

```diff
@@ -15,23 +14,23 @@ export default class ErrorDiv extends Component {
       return null
     }
 
-    let messages = {
-      "404": this.utils.objectHaveKeys(this.props, "customMessages", "404") ? this.props.customMessages["404"] : "404 Not Found",
+    // let messages = {
```

Comment:
Is there a reason why you don't use it as in https://github.com/askomics/flaskomics/blob/master/askomics/react/src/routes/error/error.jsx#L53?

Reply:
aucune, j'ai fait la modification! mais je ne vois pas trop où j'utilise cette route pour l'instant de toute manière.